README.md

```diff
@@ -61,9 +61,9 @@ end
 ?: What would the following statement return?
 
 ```ruby
-letter = "b"
+letter = 'b'
 
-if letter == "a" || "e" || "i" || "o" || "u"
+if letter == 'a' || 'e' || 'i' || 'o' || 'u'
```

Comment:
use " " for strings. single quotes don't support string interpolation and we want students to be in the habit of using syntax that won't cause bugs later on.